runHarness/AppInstance/AuctionKubernetesAppInstance.pm

```diff
@@ -155,7 +155,11 @@ override 'getEdgeAddrsRef' => sub {
 	  	 $logger->error("There are no IP addresses for the Kubernetes nodes");
 		 exit 1;
 	  }	
-      my $httpPort = $cluster->kubernetesGetNodePortForPortNumber("app=auction,type=webServer", 80, $self->namespace);
+	  # The http port will only be used if the ssl parameter is true
```

Comment:
if the ssl parameter is false?

Reply:
Good point.  in that case we should remove port 443